src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/TextBlockTests/TextBlockTests.cs

```diff
@@ -396,5 +396,23 @@ public void When_MaxLines_Changed_On_Grid_Container()
 				$"Actual Number of lines:{actualNumberOfLines}. \n" +
 				$"Line height: {lineHeight}. \n");
 		}
+
+		[Test]
+		[AutoRetry]
+		public void When_Padding_Is_Changed_Then_Cache_Is_Missed()
+		{
+			Run("UITests.Windows_UI_Xaml_Controls.TextBlockControl.TextBlock_MeasureCache");
+
+			var rect1 = _app.GetLogicalRect("textBorder");
+
+			_app.Marked("text").SetDependencyPropertyValue("Padding", "0");
+
+			var rect2 = _app.GetLogicalRect("textBorder");
+
+			_app.Marked("text").SetDependencyPropertyValue("Padding", "40");
+
+			var rect3 = _app.GetLogicalRect("textBorder");
```

Comment:
There are no assertions 😎

Reply:
I know... that's why the PR is still a draft ;-)